vue/mixins/locales.js

```diff
@@ -0,0 +1,156 @@
+        /**
+         * Gets the object with all the locales.
+         *
+         * @returns {Object} The object with all the locales.
+         */
+        getLocales() {
+            return this.$store.state.localePlugin.locales;
+        },
+        /**
+         * Appends new locales information to the specified locale. If the locale isn't
+         * specified, it will use the already chosen locale.
+         *
+         * @param {Object} locales  The locales information object. Example:
+         * { en_us: { "ripe_twitch.value_a.value_b": "Value Example" } }.
+         * @param {String} locale  The ISO-15897 standard locale definition string to be set.
+         */
+        addLocales(locales = {}, locale = null) {
+            locale = locale || this.getLocale();
+
+            const localeObj = this.getLocales()[locale] || {};
+            const updatedLocaleObj = { ...localeObj, ...locales };
+            this.setLocales({
+                ...this.getLocales(),
+                ...{ [locale]: updatedLocaleObj }
+            });
+        },
+        /**
+         * Should be called when first opening the extension. It setups the locales plugin by
+         * registering a new module in the store named "localePlugin" and sets the given locale,
+         * local locales, and fallback locale.
+         *
+         * @param {Object} store  Vuex store that is going to be used to register the needed
+         * localePlugin module.
+         * @param {Object} localLocales  The local locales information object that should be loaded
+         * and added throught this field. Example:
+         * {
+         *     en_us: { "ripe_twitch.value_a.value_b": "Value Example" },
+         *     pt_pt: { "ripe_twitch.value_a.value_b": "Valor Exemplo"}
+         * }.
+         * @param {String} locale  The ISO-15897 standard locale definition string to be set as the
+         * chosen locale.
+         * @param {String} localeFallback  The ISO-15897 standard locale definition string to be set as the
+         * fallback locale.
+         */
+        setupLocalePlugin(store, localLocales, locale, localeFallback) {
+            localLocales = localLocales || {};
+            locale = locale || Object.keys(localLocales)[0];
+            localeFallback = localeFallback || "en_us";
+
+            // Setup localePlugin store
+            store.registerModule("localePlugin", {
+                state: {
+                    locale: locale || null,
+                    localeFallback: localeFallback,
+                    locales: localLocales
+                },
+                mutations: {
+                    SET_LOCALE(state, value) {
+                        state.locale = value;
+                    },
+                    SET_LOCALE_FALLBACK(state, value) {
+                        state.localeFallback = value;
+                    },
+                    SET_LOCALES(state, value) {
+                        state.locales = value;
+                    }

```

Comment:
Use our usual style for this https://github.com/ripe-tech/ripe-commons-pluginus/blob/master/vue/store.js#L52

```suggestion
                    locale(state, value) {
                        state.locale = value;
                    },
                    localeFallback(state, value) {
                        state.localeFallback = value;
                    },
                    locales(state, value) {
                        state.locales = value;
                    }
```